### ISSUE TITLE
Cannot edit action items

### ISSUE THREAD

**Issue by felixble** [4minitz/4minitz#213]
Editing existing action items leads to an uncaught exception and the dialog does not appear. Furthermore adding a further item to the same topic will be blocked by the very same exception until the page will be reloaded.

```
Uncaught TypeError: Cannot read property 'responsibles' of undefined
    at ResponsiblePreparer._addResponsiblesFromCurrentElement (ResponsiblePreparer.js:81)
    at ResponsiblePreparer._preparePossibleResponsibles (ResponsiblePreparer.js:47)
    at ResponsiblePreparer._prepareResponsibles (ResponsiblePreparer.js:39)
    at new ResponsiblePreparer (ResponsiblePreparer.js:14)
    at configureSelect2Responsibles (topicInfoItemEdit.js:87)
    at toggleItemMode (topicInfoItemEdit.js:71)
    at String.show.bs.modal #dlgAddInfoItem (topicInfoItemEdit.js:270)
    at blaze.js?hash=f33d3df…:3775
    at Function.Template._withTemplateInstanceFunc (blaze.js?hash=f33d3df…:3744)
    at Blaze.View.<anonymous> (blaze.js?hash=f33d3df…:3774)
```

**Comment by felixble** [comment]
Unfortunately the new `ResponsiblePreparer`-class does not handle the differences between action items and topics correctly.

**Comment by felixble** [comment]
Fixed by 754769aa79f5bfb4912f2036a8be2787eb021525. Sorry I had a transposed number in the branch name as well as in the commit message...
